docs/15-mib.md

```diff
@@ -11,49 +11,47 @@ Status
 Database
 ---------------
 
-Traps are held in the database, to get rid of old ones, you can set here the number of days the traps are kept.
+Traps are held in the database. You can set the retention period for old traps here.
 
-This will occur at regular interval (not implemented in beta) or when a trap is received.
+This will occur at regular intervals (not implemented in beta) or when a trap is received.
 
-To drop some traps immediatly, you can change the number of days and click "Drop it now".
+To drop some traps manually, you can change the number of days and click "Do it now".
 
 
 Log destination
 ---------------
 
-Here is where the trap receiver will log info about what he's doing - or not doing -.
+Here is where the trap receiver will log info about what it is doing - or not doing -.
```

Comment:
```suggestion
Here is where the trap receiver will log info about what it is doing - or why he is not doing it-.
```

Reply:
I have manually added this with "it" pronoun.